SuperBuild/python_patch.cmake

```diff
@@ -4,7 +4,7 @@ set(out_msvc9compiler ${PYTHON_SRC_DIR}/Lib/distutils/msvc9compiler.py)
 
 #
 # This custom version of 'msvc9compiler.py' has been made by applying
-# the following patches to the version of 'msvc9compiler.py' shipped with Python-2.7.3.tgz:
+# the following patches to the version of 'msvc9compiler.py' shipped with Python-2.7.10.tgz:
```

Comment:
Does this mean that the file `python_patched_msvc9compiler.py` has not changed between `2.7.3` and `2.7.10` ?